tenant_schemas/test/cases.py

```diff
@@ -1,28 +1,31 @@
 from django.core.management import call_command
 from django.db import connection
-from django.test import TestCase
+from django.test import TransactionTestCase
```

Comment:
Why TransacationTestCase? And please move to a separate PR if parallel migrations does not cause this test to fail.